### PR TITLE
Add redirect resolver controlled by server

### PR DESCRIPTION
**Pull request by vldmr-k**
Add headers parameter `ts-location`, motivated by this issue #39

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ serve:
 	darkhttpd . --port 3000 --addr 127.0.0.1
 
 w:
-	gostatic -w gostatic.conf
+	./gostatic -w gostatic.conf
```
Comment (piranha) [inline Makefile:20]: ehh that's not nice :)

**Comment by piranha** [comment]
Nice, thanks! If you could revert back Makefile, that would be great 😁

**Comment by vldmr-k** [comment]
> Nice, thanks! If you could revert back Makefile, that would be great 😁

Reverted:)

**Comment by piranha** [comment]
Thanks!

**Pull request merged**